apps/example-client-app/src-tauri/src/lib.rs

```diff
@@ -10,22 +11,23 @@ pub fn run() {
     tauri::Builder::default()
         .plugin(
             tauri_plugin_log::Builder::default()
-                .level(log::LevelFilter::Warn)
+                .level(log::LevelFilter::Debug)
                 .build(),
         )
         .plugin(tauri_plugin_holochain_service_consumer::init())
         .setup(|app| {
             app.handle()
                 .holochain_service_consumer()
-                .main_window_builder(
-                    APP_ID.into(),
-                    HAPP_BUNDLE_BYTES.into(),
-                    Uuid::new_v4().to_string(),
-                    true
-                )
-                .expect("Failed to build window")
-                .build()
-                .expect("Failed to open main window");
+                .setup_app_main_window(
+                    SetupAppConfig {
+                        app_id: APP_ID.into(),
+                        happ_bundle_bytes: HAPP_BUNDLE_BYTES.into(),
+                        network_seed: Uuid::new_v4().to_string(),
+                        roles_settings: HashMap::new(),
+                        enable_after_install: true
```

Comment:
Enabling an app can fail if the `allow_deferred_memproofs` is set to true in the manifest and no memproofs are provided. So I think it's probably a problem to hard-code this here because it will make the deferred memproof case not be supported. Is there a way we can make this configurable somewhere?

Reply:
This project `apps/example-client-app` is the example demonstration of how to write a consumer app (I think we'll want to extract it into a separate repo or git template). So this is showing how it is configurable per app.

Reply:
Ah, got it :+1: